### PR TITLE
Stratum

### PR DESCRIPTION
**Pull request by AndreaLanfranchi**
Some minor optimizations.

### CODE REVIEW

File: libpoolprotocols/PoolManager.cpp [hunk 1]
```diff
@@ -89,14 +89,28 @@ PoolManager::PoolManager(PoolClient * client, Farm &farm, MinerType const & mine
...
+				cnote << string(EthYellow "Stale nonce 0x") + toHex(sol.nonce);
+			else
+				cnote << string("Nonce 0x") + toHex(sol.nonce);
+
+			p_client->submitSolution(sol);
+
+		}
+		else {
+
+			cnote << string("Nonce 0x") + toHex(sol.nonce) << "wasted. Waiting for connection ...";
```
Comment (smurfy) [inline libpoolprotocols/PoolManager.cpp]: Red or at least Yellow

Comment (AndreaLanfranchi) [reply libpoolprotocols/PoolManager.cpp]: Done red 👍

**Comment by chfast** [comment]
Please stop adding new features to release/0.14 branch.

**Comment by AndreaLanfranchi** [comment]
I can also stop contribute if you wish.


Sent from mobile. Apologies for brevity and typos.

In data 2 maggio 2018 23:06:35 Paweł Bylica <notifications@github.com> ha 
scritto:
Please stop adding new features to release/0.14 branch.

**Comment by chfast** [comment]
> I can also stop contribute if you wish.

That's not what I wish :)

**Comment by smurfy** [comment]
i would suggest and i think thats what @chfast meant is, please do the new Feature PRs against master.
Only Bugfixes against release/0.14

**Comment by AndreaLanfranchi** [comment]
Probably is my fault not being aligned on what "new feature" means.
For this commit we have :

* cb7950a which simply colorizes shares being submitted on lack of connection (several issues complain about seeing shares being submitted *after* disconnection. Maybe this is a new feature ?
* be767a8 adjusts the relase of flags before hitting handlers. Definitely not a new feature but part of the whole revised async pattern for stratum client being already part of 0.14
* 54a60de found two separate loops which could be merged into one. Not even sligtly a new feature imho.

Anyway ... I will rebase to master in the future.

**Pull request merged**